kibbeh/public/locales/de/translation.json

```
@@ -78,7 +78,7 @@
     "notFound": {
       "whoopsError": "Hoppla! Diese Seite ging im Gespräch verloren.",
       "goHomeMessage": "Keine Sorge. Du kannst",
-      "goHomeLinkText": "nach Hause"
+      "goHomeLinkText": "Nach Hause"
```

This should be lower case as it is appended to the `goHomeMessage` (I guess?)

```suggestion
      "goHomeLinkText": "nach Hause"
```